packages/compartment-mapper/test/snapshots/test-infer-exports.js.md

```diff
@@ -18,70 +18,41 @@ Generated by [AVA](https://avajs.dev).
       },
     }
 
```

Comment:
I don't understand what this file is or what I should review it for. What questions should I have in mind?

Reply:
I see what seems to be the same kind of change at https://github.com/endojs/endo/pull/1362 , so I presume this was autogenerated from redoing snapshots. Yes?

Reply:
Yep, it's a snapshot generated by an Ava test